### PR TITLE
Fix incorrect recording of net_amount

### PR DESCRIPTION
**Pull request by monishdeb**
Issue https://github.com/JMAConsulting/biz.jmaconsulting.lineitemedit/issues/59

### CODE REVIEW

**Comment by monishdeb** [comment]
@seamuslee001 can you please review the PR?

File: CRM/Lineitemedit/Upgrader.php [hunk 1]
```diff
@@ -38,4 +38,17 @@ public function upgrade_2000() {
...
+  /**
+   * Example: Run a couple simple queries.
+   *
+   * @return TRUE on success
+   * @throws Exception
+   *
+   */
+  public function upgrade_2400() {
+    $this->ctx->log->info('Applying update 2400');
+    CRM_Core_DAO::executeQuery('UPDATE civicrm_contribution SET net_amount = total_amount + fee_amount WHERE fee_amount IS NOT NULL AND fee_amount > 0');
```
Comment (MegaphoneJon) [inline CRM/Lineitemedit/Upgrader.php]: I'm pretty sure that `+` should be a `-`.

**Comment by seamuslee001** [comment]
@monishdeb we have one failed unit test that relates

```
There was 1 failure:

1) CRM_Lineitemedit_Form_AddTest::testLineItemAdditionOnBackofficeForm
Failed asserting that '0.00' matches expected 30.0.

/home/seamus/buildkit/build/47-test/web/sites/default/files/civicrm/ext/biz.jmaconsulting.lineitemedit/tests/phpunit/CRM/Lineitemedit/Form/AddTest.php:87
/home/seamus/buildkit/extern/phpunit6/phpunit6.phar:570
```

**Comment by monishdeb** [comment]
Thanks @seamuslee001 for notifying the test failure. I have fixed the issue, along some trivial change in other unit-test which were failing. Can you please check now?

**Review by seamuslee001 — APPROVED** [review]

**Comment by seamuslee001** [comment]
Changes look good to me and the Unit Tests pass now merging

**Pull request merged**